packages/plugins/type-designer/src/components/elements-type-container/content-card.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+    import Card from "@smui/card";
+    import type { DataTypeTemplates } from "./data-type-templates";
+
+    export let elementType: DataTypeTemplates[];
+
+    function handleCardClick(item: DataTypeTemplates) {
+        console.log("Clicked:", item);
+    }
+</script>
+
+<section id="type-designer-card">
+    {#each elementType as elementType}
```

Comment:
I think the loop logic should be in the `columns.svelte`component to keep this components data within its own scope (ie: columns' item)

Reply:
I don't find it necessary, since this component will only be used once, there is no need to maintain all of them as lean as possible. Leaving the loop inside makes the column component more readable

Reply:
I had in mind that we'll use the same component for the upcoming refs ? They seem to be very similar. Don't you think ?